include/z64actor.h

```diff
@@ -395,6 +402,21 @@ typedef enum {
 // Target size when activated
 #define LENS_MASK_ACTIVE_SIZE 100
 
+#define HALFDAYBIT_DAY0_DAWN  (1 << 9)
+#define HALFDAYBIT_DAY0_NIGHT (1 << 8)
+#define HALFDAYBIT_DAY1_DAWN  (1 << 7)
+#define HALFDAYBIT_DAY1_NIGHT (1 << 6)
+#define HALFDAYBIT_DAY2_DAWN  (1 << 5)
+#define HALFDAYBIT_DAY2_NIGHT (1 << 4)
+#define HALFDAYBIT_DAY3_DAWN  (1 << 3)
+#define HALFDAYBIT_DAY3_NIGHT (1 << 2)
+#define HALFDAYBIT_DAY4_DAWN  (1 << 1)
+#define HALFDAYBIT_DAY4_NIGHT (1 << 0)
+
+#define HALFDAYBIT_DAWNS      (HALFDAYBIT_DAY0_DAWN | HALFDAYBIT_DAY1_DAWN | HALFDAYBIT_DAY2_DAWN | HALFDAYBIT_DAY3_DAWN | HALFDAYBIT_DAY4_DAWN)
+#define HALFDAYBIT_NIGHTS     (HALFDAYBIT_DAY0_NIGHT | HALFDAYBIT_DAY1_NIGHT | HALFDAYBIT_DAY2_NIGHT | HALFDAYBIT_DAY3_NIGHT | HALFDAYBIT_DAY4_NIGHT)
+#define HALFDAYBIT_ALL        (HALFDAYBIT_DAWNS | HALFDAYBIT_NIGHTS)
```

Comment:
```suggestion
#define HALFDAYBIT_DAWNS  (HALFDAYBIT_DAY0_DAWN | HALFDAYBIT_DAY1_DAWN | HALFDAYBIT_DAY2_DAWN | HALFDAYBIT_DAY3_DAWN | HALFDAYBIT_DAY4_DAWN)
#define HALFDAYBIT_NIGHTS (HALFDAYBIT_DAY0_NIGHT | HALFDAYBIT_DAY1_NIGHT | HALFDAYBIT_DAY2_NIGHT | HALFDAYBIT_DAY3_NIGHT | HALFDAYBIT_DAY4_NIGHT)
#define HALFDAYBIT_ALL    (HALFDAYBIT_DAWNS | HALFDAYBIT_NIGHTS)
```
Can probably ease up on the spacing